.golangci.yml

```diff
@@ -0,0 +1,64 @@
+linters-settings:
```

Comment:
A general rule of thumb is that formatters are always better than linters and a lot of linters can do formatting now. Personally I am very soft on linting, its value to readability is overblown relative to the larger architecture choices of an app.

I would recommend configuring vscode to runs/apply formatting rules on save for you to minimize the change of committing broken formatting. If you are hotfixing prod because it is broken you don't want your builds to fail because you missed lint rule somewhere...

You can configure vscode-go to use golangci as your linter [here](https://golangci-lint.run/usage/integrations). I am also a fan of enabling [format on save](https://code.visualstudio.com/docs/languages/go#_formatting) and I normally even turn on [autoSave onFocusChange](https://code.visualstudio.com/docs/languages/go#_formatting). A common pattern would be to add these settings to your workspace rather than your user settings in vscode as well as extension recommendations. That way everyone using vscode on the project has similar base settings.